docs/contributing.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+(For now, just some notes to ick authors)
+
+## Logging
+
+We use [vmodule] to provide extra logging levels.  Use `VLOG_1` to mean "info,
+but more" and `VLOG_2` to mean "info, but even more."
+
+Debug messages should be only of interest to ick authors, whereas `VLOG_1` and
```

Comment:
I'm not sure of your exact goal with this paragraph, but it might make sense to lump the VLOGs in as a kind if info as a category? maybe

```diff
-whereas `VLOG_1`...
+whereas infos and `VLOG_1`...
```